src/logic/calculate.js

```diff
@@ -0,0 +1,27 @@
+import operate from './operate';
+
+const calculate = (calculator, buttonName) => {
+  let { total, next } = calculator;
```

Comment:
The calculator object property contains 3 properties, total, next, and operation. Let us kindly fix it